Documentation/gettingstarted/ciliumendpointslice.rst

```diff
@@ -0,0 +1,95 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gsg_ces:
+
+***************************
+CiliumEndpointSlice (beta)
+***************************
+
+.. include:: ../beta.rst
+
+This document describes CiliumEndpointSlice (CES), which enables batching of
+CiliumEndpoint (CEP) objects in the cluster to achieve better scalability.
+The concept of CiliumEndpointSlice is described in:
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   ../concepts/kubernetes/ciliumendpointslice
+
+When enabled, Cilium operator watches CEP objects and group/batch slim versions
+of them into CES objects. Cilium agent watches CES objects to learn about
+remote endpoints in this mode. API-server stress due to remote endpoint info
+propagation should be reduced in this case, allowing for better scalability,
+at the cost of potentially longer delay before identity of new endpoint is
+recognized throughout the cluster.
+
+Deploy Cilium with CES
+=======================
+The CES feature relies on use of CEP. This feature is disabled by default
+in current beta status. You can enable the feature by setting the
+``enableCiliumEndpointSlice`` value to ``true``.
+
+.. parsed-literal::
+
+  helm install cilium |CHART_RELEASE| \\
+    --set enableCiliumEndpointSlice=true
```

Comment:
Is this indented the same level as the first column character of `parsed-literal`? It looks like the indentation is not enough, which usually causes sphinx formatting issues.

Reply:
Done.